### ISSUE TITLE
Accessibility Feedback - Design Intent: eBenefits - View Dependents Formlet

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21464]
## Feedback
- **Must** = the feedback must be applied
- **Should** = the feedback is best practice
- **Consider** = suggestions/enhancements

**Must:**
* Have accessible labels for the buttons
  * "Remove this dependent, Pat Smith" or
  * "Remove this dependent, spouse"
  * "Cancel removing spouse"
* Announce the green status box to screen readers. This can be accomplished by treating the container as a `role="status"` or setting keyboard focus on the container. If the container disappears after a set time, I'd recommend treating it as a status message or live region, otherwise, focus will be lost when it disappears.
* Manage focus when you switch into edit-in-place from the read-only view, and when you return to read-only.
  * Might consider setting focus on the person's name (heading) when you exit edit-in-place.
  * Focus could be set on the instruction text "To remove this dependent..." when you enter edit-in-place.

**Should:**
* Annotate focus management when local state changes. By being intentional and handing this off as a spec to engineering, we can simplify the testing cycle and validate that we're announcing the right information to assistive tech sooner.

**Consider:**
* Slides 3 and 4 of 9: The first radio button says "Dependent stepchild has left household". Should this ask about child and stepchild?

**Comment by tblackwe** [comment]
This is feedback from collab cycle. I feel this can be closed out as the feature that it applied to will have been resolved already.

**Comment by tblackwe** [comment]
These accesibility changes were not made previously. accepting this into our backlog.

**Comment by sortizsh** [comment]
@juliepedtke + Daniela can you take a look?

**Comment by tblackwe** [comment]
This will be re-reviewed as part of accessibility audit. we can hold off until that is complete.

**Comment by tblackwe** [comment]
@ohgreatkate: I've asked @zack to review this ticket on our side.

**Comment by zack** [comment]
@tblackwe @sortizsh @juliepedtke 

Tyler and I both looked at this and need some more information before we can make any confident statements about the state of the work. We can't figure out where this "edit-in-place" formlet is from either the code or clicking around the website. We also don't have access to the referenced slide deck.

Ideally we can get screenshots of the relevant features, but barring that, some strings we can grep for might be enough.

**Comment by tblackwe** [comment]
@zack @tfink419 did you find anything about this formlet in the code? I have a suspicion that the formlet wasnt implemented, since I only see removing dependents in the add/remove flow

**Comment by tblackwe** [comment]
also, I didnt see reference of a slide deck, I can check on getting access if you can point me to the link

**Comment by zack** [comment]
The last line of the ticket references "Slides 3 and 4 of 9," but there is no link.

**Comment by tblackwe** [comment]
it sure does. I was looking for a link. Thanks! Further evidence that this ticket might not be workable due to age.

**Comment by tblackwe** [comment]
closing this ticket. we cant find the documentation or place in application it exists. closing this under assumption this design was never implemented.
